modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java

```diff
@@ -17,7 +17,13 @@
 import com.liferay.analytics.reports.info.item.ClassNameClassPKInfoItemIdentifier;
 import com.liferay.info.item.ClassPKInfoItemIdentifier;
 import com.liferay.info.item.InfoItemReference;
+<<<<<<< HEAD
```

Comment:
Be careful with this conflict lines. They shouldn't be here

Reply:
@Stephy6 let's look at this with @claraizquierdo  or with me, we will show you how to properly solve conflicts ;)

Reply:
Done! thanks @claraizquierdo !